server/models/Collective.js

```diff
@@ -1186,21 +1186,26 @@ function defineModel() {
 
   Collective.prototype.getChildren = function (query = {}) {
     return Collective.findAll({
+      order: ['createdAt', 'DESC'],
```

Comment:
Following up on https://github.com/opencollective/opencollective/issues/4598 I want to update all places where we're ordering on non-unique fields to make sure we have a secondary condition. Though rare, it's not impossible to end up with transactions or collectives that have the exact same `createdAt`. In these cases, the order is unpredictable and can cause pagination issues.

The solution is quite simple, add a secondary condition on `id`. Here for example:
```suggestion
      order: [['createdAt', 'DESC'], ['id', 'DESC']],
```

Here there's almost no chance to see a problem happening (you don't create two projects at the exact same time), so no requested change.

In `getEvents`, though pretty edge case, it's also conceivable to have two events happening at the same time for a collective. We could make sure the order stays predictable with something like:

```es6
      order: [
        ['startsAt', 'DESC'],
        ['endsAt', 'DESC'],
        ['id', 'DESC'],
      ],
```